charts/argocd-apps/Chart.yaml

```diff
@@ -17,5 +17,5 @@ annotations:
     fingerprint: 2B8F22F57260EFA67BE1C5824B11F800CD9D2252
     url: https://argoproj.github.io/argo-helm/pgp_keys.asc
   artifacthub.io/changes: |
-    - kind: deprecated
-      description: Add deprecated note for Argo CD Extensions
+    - kind: added
+      description: ability to specify ignoreApplicationDifferences for applicationsets
```

Comment:
You need new line in the end to pass CI lint 🙋 

https://github.com/argoproj/argo-helm/pull/2442/files/dc324fd474e2c785f25dc9ce1bd49b270b54189f..a53d032ae8f7974701a8d41b8239853691f1489f

Reply:
Also please bump version, maybe you missied it when you fetch remote latest diff.